application/controllers/SettingsController.php

```diff
@@ -422,13 +422,18 @@ private function checkSnmpTrapd()
   {
       $psOutput=array();
       // First check is someone is listening to port 162. As not root, we can't have pid... 
-      exec('netstat -an |grep -E "udp.*:162"',$psOutput);
+      $sspath = '/usr/sbin/ss';
+      if(!is_executable("$sspath"))
+      {
+          return array(1,"Can not execute $sspath");
+      }
+      exec("$sspath -luna | grep ':162 '",$psOutput);
```

Comment:
Remove the -a option to get only listening socket
```suggestion
      exec("$sspath -lun | grep ':162 '",$psOutput);
```